data-index/data-index-service/data-index-service-common/src/test/java/org/kie/kogito/index/service/AbstractIndexingServiceIT.java

```diff
@@ -794,6 +795,17 @@ void testUserTaskInstanceIndex() throws Exception {
                 .body("data.Deals[0].metadata.userTasks[0].started", is(formatZonedDateTime(event.getData().getStarted().withZoneSameInstant(ZoneOffset.UTC))))
                 .body("data.Deals[0].metadata.userTasks[0].completed", is(formatZonedDateTime(event.getData().getCompleted().withZoneSameInstant(ZoneOffset.UTC))))
                 .body("data.Deals[0].metadata.userTasks[0].lastUpdate", is(formatZonedDateTime(event.getTime().withZoneSameInstant(ZoneOffset.UTC))));
+
+        event.setData((new KogitoUserTaskCloudEventDeserializer().deserialize(null, readFileContent("user_task_instance_event.json")
```

Comment:
Wouldnt be enought to do:
```suggestion
        event.getData().setActualOwner("admin");
```

Reply:
it does not work, because of https://github.com/kiegroup/kogito-apps/blob/360478879e2d8086707c8d9ee3498883021c8595/data-index/data-index-storage/data-index-storage-api/src/main/java/org/kie/kogito/index/model/UserTaskInstanceMeta.java#L86

Reply:
this should help:
```suggestion
        event = getUserTaskCloudEvent(taskId, processId, processInstanceId, null, null, state);
        event.getData().setActualOwner(null);
```

Reply:
Done

Reply:
Done